woocommerce/woocommerce
# deprecate wc-setup

### All Submissions:


### Changes proposed in this Pull Request:

This PR deprecates the wc-setup page functionality. It will require some [changes in WooCommerce Admin](https://github.com/woocommerce/woocommerce-admin/issues/4575) to replace redirects, settings, etc.

Note: E2E tests fail due to #4 in test steps.

Closes #26173 .
Closes #27286 .

### How to test the changes in this Pull Request:

1. Create a new WP install
2. Activate this branch
3. Check that the four WC pages are created
4. Activating should not redirect to the onboarding splash page (to be added by WC Admin)
5. The help popup should not have the `Setup` tab.

### Other information:




### Changelog entry

> Deprecate old setup wizard.
> Add WC pages on installation.
